tests/test.rs

```diff
@@ -669,8 +669,8 @@ fn test_parse_bool() {
             ("truz", "expected ident at line 1 column 4"),
             ("f", "expected ident at line 1 column 1"),
             ("faz", "expected ident at line 1 column 3"),
-            ("truea", "trailing characters at line 1 column 5"),
-            ("falsea", "trailing characters at line 1 column 6"),
```

Comment:
This PR changes these error messages. Let me know if that's a problem, although in my opinion the new error is more accurate